docs/source/decisions/0002-data-versioning.rst

```diff
@@ -0,0 +1,66 @@
+##################################
+Data Types use Calendar Versioning
+##################################
+
+Summary
+=======
+
+Instead of versioning the entire API, we version the data types if appropriate.
+
+Opinions
+========
+
+Opinion 1
+---------
+
+Summary: If a given end point can have additional, or modified, elements in the response, add a data version.
+   
+
+@MikeNeilson
+
+By versioning the data, and using the Content-Type and Accept headers and the full features of MIME types we appropriately 
+separate the concern of "what data we are retrieving/storing" from the presentation of data.
+
+e.g /timeseries/Alder Springs.Temp-Air.Inst.15Minutes.0.GOES-raw?begin=PT0&end=PT-1D&units=C, granted with a reasonable
+ exception to the units, defines *what* we want.
+
+The header, Accept, informs the API what format, or formats, we are willing to accept the data in. 
+
+
+
+Decision Status
+---------------
+
+partial acceptance
+
+Data, by content-types, are versioned. In the past there was some severe confusion on this part and it was treated as anything 
+new was "version=2" in the content-type. To allow this design but reduce confusion going forward
+
+1. The initial content-type of a data set *SHALL* be be the plain content-type and *SHOULD* include an additional expanded content-type
+3. *IF* is it not the first version of this data, additional information will be set in the content-type as 
+   appropriate to the to the data. (e.g. `application/json+<something>` or `application/json;<something>`)
```

Comment:
I think this needs a more concrete example than `<something>`, i.e. to demonstrate the problem we are trying to solve